Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -108,13 +108,11 @@ Common::Symbol* PPCSymbolDB::GetSymbolFromAddr(u32 addr)
   return nullptr;
 }
 
-std::string PPCSymbolDB::GetDescription(u32 addr)
+std::string_view PPCSymbolDB::GetDescription(u32 addr)
```

Comment:
Not really something that needs to change in this PR but surprised this isn't `const`

Reply:
Yep, `PPCSymbolDB` and its base class, `Common::SymbolDB`, lack a lot of const-correctness as technical debt from them being global variables for so many years. I can turn this PR into a general "PPCSymbolDB updates" PR if that's desired, I just saw this as a quick and simple fix to get approved fast.

Reply:
Nah, can be saved for another day.  Code as is looks good.  Thanks @mitaclaw